### DISCUSSION TITLE
使用ST LINK下载是提示Error: File does not exist: EIDE

### DISCUSSION THREAD

**Discussion by warrior2545** [github0null/eide#281] [Q&A]
操作系统版本： win11 22H2
操作系统平台(x86/x64)：X64
VsCode 版本：1.81.0
EIDE 插件版本：v3.13.2023060401
C/C++ 插件版本：v1.17.4
何种编译器(keil_c51/sdcc/armcc5/armgcc/...)：armcc5
编译器版本(非编译问题可忽略)：
描述问题

工程可以正常编辑 但是无法完成烧录

屏幕截图

https://discuss.em-ide.com/d/579-st-linkerror-file-does-not-exist-eide

期望现象

可以正常完成 下载烧录操作
一人研究至深夜凌晨还是没能解决问题求大佬 伸手援助 真的很喜欢用EIDE 只差最后一公里打通

**Comment by github0null** [comment]
https://discuss.em-ide.com/d/579-st-linkerror-file-does-not-exist-eide/4
